resources/sync_gateway_configs/bucket_online_offline/bucket_online_offline_default_di.json

```diff
@@ -10,16 +10,20 @@
     "cluster_config": {
         "server":"http://{{ couchbase_server_primary_node }}:8091",
         "data_dir":".",
-        "bucket":"data-bucket"
+        "bucket":"data-bucket",
+        "password": "password"
     },
     "databases":{
         "db":{
             "feed_type":"DCPSHARD",
             "server":"http://{{ couchbase_server_primary_node }}:8091",
             "bucket":"data-bucket",
+            "password": "password",
             "channel_index":{
                 "server":"http://{{ couchbase_server_primary_node }}:8091",
                 "bucket":"index-bucket",
+                "username":"index-bucket",
```

Comment:
Should we file an inconsistency bug in behavior between data / index buckets. Sometimes a username is required and sometime it is not. This is a bit confusing.